gtsfm/scene_optimizer.py

```diff
@@ -231,15 +231,16 @@ def _filter_dict_keys(dict: Dict[Any, Any], ref_dict: Dict[Any, Any]) -> Dict[An
         # Save metrics to JSON and generate HTML report.
         auxiliary_graph_list.extend(save_metrics_reports(metrics_graph_list))
 
-        # # Modify BA input and BA output to have point clouds and frustums aligned with x,y,z axes.
-        # ba_input_graph, ba_output_graph, gt_pose_graph = dask.delayed(align_estimated_gtsfm_data, nout=3)(
-        #     ba_input_graph, ba_output_graph, gt_pose_graph
-        # )
+        # Modify BA input, BA output, and GT poses to have point clouds and frustums aligned with x,y,z axes.
+        gt_poses_graph = (
```

Comment:
needed to move `gt_poses_graph` up in the code so that the function call to `align_estimated_gtsfm_data` has access to it